nvtabular/inference/triton/model.py

```diff
@@ -48,6 +48,19 @@
 class TritonPythonModel:
     """Generic TritonPythonModel for nvtabular workflows"""
 
+    def _initialize_ops(self, column_group, initialized):
+        if column_group.op:
+            inference_op = column_group.op.inference_initialize(
+                column_group.columns, self.model_config
+            )
+            if inference_op:
+                column_group.op = inference_op
+
+        for parent in column_group.parents:
+            if parent not in initialized:
+                initialized.add(parent)
```

Comment:
The name of the `initialized` parameter is a little confusing, because the `column_group`'s op hasn't actually been initialized until after its added to the `initialized` set. Maybe something like `visited` would be clearer here?

Reply:
I suppose the problem with giving it a default value is that the default is only evaluated once resulting in a single mutable set being shared across invocations. Could the default be `None` and then the body of the method would create an empty set to use if so? That would make it a little easier to use from the perspective of the calling code.

Reply:
done